tests/integration/security/reachability_test.go

```diff
@@ -34,6 +34,7 @@ import (
 // - Send HTTP/gRPC requests between apps.
 func TestReachability(t *testing.T) {
 	framework.NewTest(t).
+		RequiresEnvironment(environment.Kube).
```

Comment:
Why are these needed? Is it a bug in upstream tests? If yes, could you please send PR's there (I'll approve them)